website/www/site/content/en/documentation/ml/about-ml.md

```diff
@@ -112,14 +112,19 @@ that illustrates running Scikit-learn models with Apache Beam.
 
 #### TensorFlow
 
-To use TensorFlow with the RunInference API, you need to do the following:
-
-* Use `tfx_bsl` version 1.10.0 or later.
-* Create a model handler using `tfx_bsl.public.beam.run_inference.CreateModelHandler()`.
-* Use the model handler with the [`apache_beam.ml.inference.base.RunInference`](/releases/pydoc/current/apache_beam.ml.inference.base.html) transform.
-
-See [this notebook](https://github.com/apache/beam/blob/master/examples/notebooks/beam-ml/run_inference_tensorflow.ipynb)
-that illustrates running TensorFlow models with Apache Beam and tfx-bsl.
+To use TensorFlow with the RunInference API, you have two options:
```

Comment:
I think we need to provide reasons on why you would use each. Specifically: the beam model handlers accept numpy/tensors, the tfx_bsl one requires tf.examples

Reply:
Same comment applies both places